<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookSystemClass</class>
 <widget class="QMainWindow" name="BookSystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BookSystem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>641</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="search">
     <attribute name="title">
      <string>检索</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>24</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>书名</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_select">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>611</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_serach">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string> 查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_serach_empty">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>15</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>仅显示有余量</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>15</y>
        <width>24</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>类别</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>15</y>
        <width>24</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>作者</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>默认</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>类别</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>作者</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>书名</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>反向</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>24</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>排序</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <zorder>lineEdit</zorder>
     <zorder>label</zorder>
     <zorder>tableView_select</zorder>
     <zorder>pushButton_serach</zorder>
     <zorder>pushButton_serach_empty</zorder>
     <zorder>checkBox</zorder>
     <zorder>label_2</zorder>
     <zorder>lineEdit_2</zorder>
     <zorder>lineEdit_3</zorder>
     <zorder>label_3</zorder>
     <zorder>gridLayoutWidget</zorder>
     <zorder>label_4</zorder>
     <zorder>checkBox_2</zorder>
     <zorder>plainTextEdit</zorder>
    </widget>
    <widget class="QWidget" name="insert">
     <attribute name="title">
      <string>入库</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>借出</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="card">
     <attribute name="title">
      <string>借阅信息</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BookSystem.qrc"/>
 </resources>
 <connections/>
</ui>
